libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -554,6 +554,104 @@ bool AP_Mount_Backend::get_angle_target_to_sysid(MountTarget& angle_rad) const
     return get_angle_target_to_location(_target_sysid_location, angle_rad);
 }
 
+bool AP_Mount_Backend::get_target(float& roll_deg, float& pitch_deg, float& yaw_deg)
```

Comment:
Shouldn't this be sharing code with the update function?!

Reply:
@peterbarker . If you are concerned with the update function in every individual backend type I think it can't share code with with that. May be you could provide a hint.